### PR TITLE
New route - Blog

### LINKED ISSUE #40: Add a "Blog" section
This section will feature all of the release notes and other updates.
It will have a sidebar that will be built based on some json configuration (like our examples and docs).
Each post should have a dedicated MDX file.

### PR DESCRIPTION
**Pull request by davidyushkov**
added blog page with mdx content and sidebar
webrix_blog

### CODE REVIEW

File: src/routes/Blog/Blog.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import routes from './Blog.routes';
+import {Container} from 'components';
+import Sidebar from './components/Sidebar/Sidebar';
+import Content from './components/Content/Content'
+import './Blog.scss';
+
+const Blog = () => (
+    <>
+        <Helmet>
+            <title>Blog</title>
```
Comment (ykadosh) [inline src/routes/Blog/Blog.jsx]:
```suggestion
            <title>Blog - Webrix.js</title>
```

File: src/routes/Blog/Blog.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            <meta name='description' content='webrix blog posts'/>
+        </Helmet>
+        <Container>
+            <Sidebar/>
+            <Content/>
+        </Container>
+        <Route>
+            <Redirect to={routes[0].path}/>
```
Comment (ykadosh) [inline src/routes/Blog/Blog.jsx:21]: Why is this needed?

**Review by ykadosh — CHANGES_REQUESTED** [review]

File: src/routes/Blog/Blog.jsx (hunk 2 above)
Comment (davidyushkov) [reply src/routes/Blog/Blog.jsx:21]: In case if user is going to /blog to get the first post from the blog and not to render blank page. The same thing is done for Examples.

**Review by ykadosh — APPROVED** [review]
